DEVELOPMENT.md

```diff
@@ -187,18 +185,66 @@ pare summarised below:
 * EMAIL_CONFIG: configuration of the SMTP email server that is used for email verification. It also allows users to retrieve a new password after forgetting it. Don't forget to enter the reply address (REPLY_ADDRESS) of your system emails. Omit this parameter if system emails for verification and password retrieval are unwanted.
 * OAUTH: an authenticated ASReview application may integrate with the OAuth functionality of Github, Orcid and Google. Provide the necessary OAuth login credentails (for [Github](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app), [Orcid](https://info.orcid.org/documentation/api-tutorials/api-tutorial-get-and-authenticated-orcid-id/) en [Google](https://support.google.com/cloud/answer/6158849?hl=en)). Please note that the AUTHORIZATION_URL and TOKEN_URL of the Orcid entry are sandbox-urls, and thus not to be used in production. Omit this parameter if OAuth is unwanted.
 
-### Converting an unauthenticated application in an authenticated one
+### Converting an unauthenticated application into an authenticated one
+
+Start the application with authentication enabled for the first time. This ensures the creation of the necessary database. To avoid unwanted user input, shutdown the application.
+
+To convert the old unauthenticated projects into authenticated ones, the following steps should be taken:
+
+1. Create user accounts for people to sign in.
+2. Convert project data and link the projects to the owner's user account.
+
+Under the CLI sub commands of the ASReview application a tool can be found that facilitates these procedures:
+
+```
+$ asreview auth-tool --help
+```
+
+#### Creating user accounts
+
+The first step is to create user accounts. This can be done interactively or by using a JSON string to bulk insert the accounts. To add user accounts interactively run the following command:
+```
+$ asreview auth-tool add-users --interactive --db-path ~/.asreview/asreview.development.sqlite
```

Comment:
```suggestion
$ asreview auth-tool add-users --interactive --db-path ~/.asreview/asreview.production.sqlite
```

Who is the intended audience here? Developers or admins?

Reply:
fixed